addons/account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -62,9 +62,10 @@ def import_file(self):
         return {
             'type': 'ir.actions.client',
             'tag': 'bank_statement_reconciliation_view',
-            'context': {'statement_line_ids': statement_line_ids_all,
-                        'company_ids': self.env.user.company_ids.ids,
-                        'notifications': notifications_all,
+            'context': {
+                'statement_line_ids': statement_line_ids_all,
+                'company_ids': self.env.companies.ids, # VFE FIXME allowed companies or user companies for bank statement import ?
```

Comment:
@smetl when importing bank statements, should we always fallback on all companies of user, or only those in the environment ?